src/components/Count/CountTag.tsx

```diff
@@ -0,0 +1,35 @@
+import { Typography } from '@material-ui/core';
```

Comment:
looks good!! if you could make Playlist Page + SeeAll pages use this component as well that'd be lovely!!